cmake/toolchain/specific.json

```diff
@@ -1,7 +1,6 @@
 {
   "unit.meta.exe":  "^unit.meta\\..*\\.exe",
   "unit.arch.exe":  "^unit.arch\\..*\\.exe",
-  "unit.core.bit_cast.exe":  "^unit.core.bit_cast.exe",
```

Comment:
I don't understand - you just moved it, right?

Reply:
look down the file, it's there.